src/tools/compiletest/src/runtest.rs

```diff
@@ -2403,22 +2404,26 @@ impl<'test> TestCx<'test> {
         ];
         let tidy_dir = |dir| {
             let tidy = |file: &_| {
-                Command::new("tidy")
-                    .args(&tidy_args)
-                    .arg(file)
-                    .spawn()
-                    .unwrap_or_else(|err| {
-                        self.fatal(&format!("failed to run tidy - is it installed? - {}", err))
-                    })
-                    .wait()
-                    .unwrap()
+                let tidy_proc = Command::new("tidy").args(&tidy_args).arg(file).spawn();
```

Comment:
Instead of doing this check in the diff, wouldn't it be better to do it before and then having the error message print only once?

Reply:
Good idea, done.